cli/cook/subcommands/usage.py

```diff
@@ -0,0 +1,172 @@
+import json
+
+import sys
+
+from cook import http, colors
+from cook.format import format_job_memory
+from cook.querying import query_across_clusters, make_job_request
+from cook.util import guard_no_cluster, current_user, print_info
+
+
+def get_usage_on_cluster(cluster, user):
+    """Queries cluster for jobs with the given state / user / time / name"""
+    params = {'user': user, 'group_breakdown': 'true'}
+    usage_map = http.make_data_request(cluster, lambda: http.get(cluster, 'usage', params=params))
+    if not usage_map:
+        raise Exception(f'Unable to retrieve usage information on {cluster["name"]} ({cluster["url"]}).')
+
+    share_map = http.make_data_request(cluster, lambda: http.get(cluster, 'share', params={'user': user}))
+    if not share_map:
+        raise Exception(f'Unable to retrieve share information on {cluster["name"]} ({cluster["url"]}).')
```

Comment:
It might be good to catch these two exceptions somewhere within this module so that you can report partial results even if there's an error contacting one of your configured clusters. Obviously we still want to report the error—but maybe it shouldn't clobber everything else.